.github/workflows/ci_cd.yml

```diff
@@ -124,26 +125,26 @@ jobs:
       - get-changes
 
     steps:
-      - name: Checkout repository
-        uses: actions/checkout@v3
-
-      - name: Add API stack label
-        uses: actions-ecosystem/action-add-labels@v1
-        if: ${{ needs.get-changes.outputs.api == 'true' }}
-        with:
-          labels: "🧱 stack: api"
-
-      - name: Add ingestion server stack label
-        uses: actions-ecosystem/action-add-labels@v1
-        if: ${{ needs.get-changes.outputs.ingestion_server == 'true' }}
-        with:
-          labels: "🧱 stack: ingestion server"
-
-      - name: Add frontend stack label
-        uses: actions-ecosystem/action-add-labels@v1
-        if: ${{ needs.get-changes.outputs.frontend == 'true' }}
+      - uses: actions/github-script@v6
         with:
-          labels: "🧱 stack: frontend"
+          # Also see
+          # - list of stack filters: `.github/filters.yml`
+          # - list of stack labels: https://github.com/WordPress/openverse/labels?q=stack
+          script: |
```

Comment:
Is this a step to add labels to PRs? Cool that this can be simplified and a dependency removed! 💯 (`actions-ecosystem/action-add-labels@v1`)

Would you mind naming it?

Reply:
Yeah, this was great and removed the need for all the individual steps. Added the name in 3beb987.